lib/phoenix_live_view/html_algebra.ex

```diff
@@ -27,7 +27,6 @@ defmodule Phoenix.LiveView.HTMLAlgebra do
       {_type, next_doc} = to_algebra(node, context)
       concat(doc, next_doc)
     end)
-    |> force_unfit()
```

Comment:
This was forcing the line break. I guess it is fine to live without it since all the tests I fixed, make sense to be inline since they do fit in the line length.